src/Features/CSharp/Portable/Completion/CompletionProviders/AttributeNamedParameterCompletionProvider.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
 {
-    internal class AttributeNamedParameterCompletionProvider : CommonCompletionProvider
+    internal class AttributeNamedParameterCompletionProvider : CommonCompletionProvider, IFeaturesCustomCommitCompletionProvider
```

Comment:
why?  this doesn't need any custom comitting behavior.

Reply:
I'm going to remove IFeaturesCustomCommitCompletionProvider. All provider will support this.

---
In reply to: [222523875](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523875) [](ancestors = 222523875)